staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -92,7 +93,8 @@ type crdHandler struct {
 	// atomic.Value has a very good read performance compared to sync.RWMutex
 	// see https://gist.github.com/dim/152e6bf80e1384ea72e17ac717a5000a
 	// which is suited for most read and rarely write cases
-	customStorage atomic.Value
+	customStorage         atomic.Value
+	previousCustomStorage sync.Map
```

Comment:
I like the ability to store the prev_version of a CRD like this to be able to distinguish a v0 -> v1 update from v2 -> v1 update. But I am also concerned that this means that we will process SV updates only when new CRDInfo is created - which is done whenever a CR request is received.

On CRD updates however, we will only be updating the previousCustomStorage map and no SV updates are done. Is this okay?